### ISSUE TITLE
Cannot enumerate named pipes with Get-ChildItem

### ISSUE THREAD

**Issue by jborean93** [PowerShell/PowerShell#11898]
# Steps to reproduce

On Windows run the following on PowerShell Core 7

```powershell
# Works on 5, 6, but not 7
Get-ChildItem -Path '\\.\pipe\' | Select-Object -ExpandProperty FullName

# Works on all
[System.IO.Directory]::EnumerateFiles('\\.\pipe\')
```

# Expected behavior

`Get-ChildItem` should return the named pipes on all PowerShell versions and match the output of the .NET method.

# Actual behavior

PowerShell 5 and 6 output the named pipes but 7 outputs the FullName of contents of the root drive.

```none
PS C:\Users\vagrant-domain> Get-ChildItem -Path '\\.\pipe\' | Select-Object -ExpandProperty FullName
C:\PerfLogs
C:\Program Files
C:\Program Files (x86)
C:\temp
C:\tools
C:\Users
C:\Windows

PS C:\Users\vagrant-domain> [System.IO.Directory]::EnumerateFiles('\\.\pipe\')
\\.\pipe\InitShutdown
\\.\pipe\lsass
\\.\pipe\ntsvcs
\\.\pipe\scerpc
...
```

Even using `-LiteralPath` does not work on v7 and `Get-Item -Path '\\.\pipe\` (with and without the trailing \) fails to find a file

# Environment data

```none
# PS v5 (works)
Name                           Value
----                           -----
PSVersion                      5.1.17763.771
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.17763.771
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

# PS v6 (works)
Name                           Value
----                           -----
PSVersion                      6.2.4
PSEdition                      Core
GitCommitId                    6.2.4
OS                             Microsoft Windows 10.0.17763
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

# PS v7 (doesn't work)
Name                           Value
----                           -----
PSVersion                      7.0.0-rc.2
PSEdition                      Core
GitCommitId                    7.0.0-rc.2
OS                             Microsoft Windows 10.0.17763
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by zhaohansprt** [comment]
what now is the solution

**Comment by AndrewShepherd** [comment]
This issue is fixed in Powershell version 7.5.
